metriq_gym/process.py

```diff
@@ -28,10 +28,10 @@
 
 def get_job(result: BenchJobResult) -> Job:
```

Comment:
Should these functions live in the `JobManager` class instead of in `process.py`?